source/js/renderer.js

```diff
@@ -251,7 +253,8 @@ Renderer.prototype.getPageHit = function (clientX, clientY) {
     clientY -= bounds.top;
 
     // check each rendered page to see if the x,y are inside the page bounds
-    for (var i = 0; i < this._renderedPages.length; i++) {
+    for (var i = 0; i < this._renderedPages.length; i++)
```

Comment:
You shouldn't constantly check the length, since it's not going to change in this loop. This is better:

```javascript
for (var i = 0, len = this._renderedPages.length; i < len; i++)
{
 ...
}
```